### PR TITLE
[TD] Fix problem with translation

### PR DESCRIPTION
**Pull request by Kuzma30**
Switch from QT_TRANSLATE_NOOP to QtGui.QApplication.translate.
This solves issues 145-147 from https://github.com/FreeCAD/FreeCAD-translations/issues

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6931">pipeline status</a> for feature branch [PR_6931](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6931). Pipeline [550514301](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/550514301) was triggered at [77f8e54](https://github.com/FreeCAD/FreeCAD/pull/6931/commits/77f8e545ca74d777624f0eb763e5777e7aaba426). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
CC @yorikvanhavre does this look correct?

**Comment by Kuzma30** [comment]
https://github.com/FreeCAD/FreeCAD-translations/issues/147
Here is problem description.

File: src/Mod/TechDraw/TechDrawTools/CommandShareView.py [hunk 1]
```diff
@@ -26,7 +26,8 @@
 __version__ = "00.01"
 __date__ = "2022/01/11"
 
-from PySide.QtCore import QT_TRANSLATE_NOOP
+from PySide import QtCore
```
Comment (benj5378) [inline src/Mod/TechDraw/TechDrawTools/CommandShareView.py:29]: Why is `from PySide import QtCore`? Isn't it unused?
```suggestion
```

File: src/Mod/TechDraw/TechDrawTools/CommandMoveView.py [hunk 2]
```diff
@@ -26,7 +26,8 @@
 __version__ = "00.01"
 __date__ = "2022/01/11"
 
-from PySide.QtCore import QT_TRANSLATE_NOOP
+from PySide import QtCore
```
Comment (benj5378) [inline src/Mod/TechDraw/TechDrawTools/CommandMoveView.py:29]: Why is `from PySide import QtCore`? Isn't it unused?
```suggestion
```

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6931">pipeline status</a> for feature branch [PR_6931](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6931). Pipeline [550663471](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/550663471) was triggered at [1e9ed1e](https://github.com/FreeCAD/FreeCAD/pull/6931/commits/1e9ed1e6c99e784f4a2402f12ef4c322400001e4). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by Kuzma30** [comment]
@benj5378 Thanks for review

**Comment by donovaly** [comment]
@Kuzma30 , please let the CI run over your PRs. There can always be a typo and the CI will find it.

File: src/Mod/TechDraw/TechDrawTools/CommandMoveView.py [hunk 3]
```diff
@@ -44,8 +44,8 @@ def GetResources(self):
         """Return a dictionary with data that will be used by the button or menu item."""
         return {'Pixmap': 'actions/TechDraw_MoveView.svg',
                 'Accel': "",
-                'MenuText': QT_TRANSLATE_NOOP("MoveView", "Move View"),
-                'ToolTip': QT_TRANSLATE_NOOP("MoveView", "Move a View to a new Page")}
+                'MenuText': QtGui.QApplication.translate("MoveView", "Move View", None),
+                'ToolTip': QtGui.QApplication.translate("MoveView", "Move a View to a new Page", None)}
```
Comment (chennes) [inline src/Mod/TechDraw/TechDrawTools/CommandMoveView.py:48]: This change isn't correct: menu text and tooltips should use NOOP to add them to the translation database. The actual translation occurs elsewhere.

**Review by chennes — CHANGES_REQUESTED** [review]
The changes to the menu text and tooltip are incorrect, and throughout the file I believe the context is wrong for all translations. The name of the command is "TechDraw_MoveView", not just "MoveView".

**Comment by Kuzma30** [comment]
@yorikvanhavre @chennes May be I recreate PR? I think only change context (add TechDraw_ suffix) All other thing may be as is.

**Comment by chennes** [comment]
Yeah, that's probably a good idea.

**Pull request closed without merging**